runtime/bcutil/ROMClassBuilder.cpp

```diff
@@ -1059,17 +1059,17 @@ ROMClassBuilder::finishPrepareAndLaydown(
  *                                  + UNUSED
  *                                 + UNUSED
  *                                + UNUSED
- *                               + UNUSED
+ *                              + J9AccClassHiddenOptionNestmate
  *
- *                             + UNUSED
+ *                             + J9AccClassHiddenOptionStrong
  *                            + AccSealed
  *                           + AccRecord
  *                          + AccClassAnonClass
  *
  *                        + AccSynthetic (matches Oracle modifier position)
  *                       + AccClassUseBisectionSearch
  *                      + AccClassInnerClass
- *                     + UNUSED
+ *                     + J9AccClassHiddenClass
```

Comment:
The trailing `Class` here seems superfluous.

Reply:
I'd still like this changed.

Reply:
> I'd still like this changed.

Yes, I will change this. But after lunch.

Reply:
Fixed.